### PR TITLE
feat(dashmate)!: handle already configured certificate in obtain command

### PR DESCRIPTION
**Pull request by shumkov**
## Issue being fixed or feature implemented

The SSL certificate obtain command creates a new certificate even if one is already created. 

## What was done?

1. Download Zero SSL certificate if it's already configured with the obtain command
2. Continue certificate validation process with the obtain command if the existing certificate requires verification
3. Renew existing certificate with the obtain command if it's about to expire
4. Provide a `force` flag to the obtain command to create a new certificate even if one is already configured
5. Added revoke and cancel certificate functions (used to remove 200+ certificates from our account 🫣)

## How Has This Been Tested?

Manually

## Breaking Changes

- Removed `dashmate ssl renew` command which duplicates `dashmate ssl obtain` 

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

**Review by jawid-h — DISMISSED** [review]

File: packages/dashmate/configs/schema/configJsonSchema.js [hunk 1]
```diff
@@ -368,8 +368,12 @@ module.exports = {
                               type: ['string', 'null'],
                               minLength: 32,
                             },
+                            privateKey: {
+                              type: ['string', 'null'],
+                              minLength: 32,
```
Comment (strophy) [inline packages/dashmate/configs/schema/configJsonSchema.js:373]: This string in PEM format is many hundreds of characters, will we store it in the config here?

File: packages/dashmate/src/commands/ssl/obtain.js [hunk 2]
```diff
@@ -50,13 +55,20 @@ class ObtainCommand extends ConfigBaseCommand {
...
+In case if certificate is about to expire (see 'expiration-days' flag) it will be renewed
 `;
 
 ObtainCommand.flags = {
   ...ConfigBaseCommand.flags,
   verbose: Flags.boolean({ char: 'v', description: 'use verbose mode for output', default: false }),
   'no-retry': Flags.boolean({ description: 'do not retry on IP verification failure', default: false }),
+  force: Flags.boolean({ description: 'renew even if certificate is valid', default: false }),
+  'expiration-days': Flags.integer({
+    description: 'renew even expiration period is less than'
```
Comment (strophy) [inline packages/dashmate/src/commands/ssl/obtain.js:68]: even if

**Review by strophy — COMMENTED** [review]
Looks like a much more robust design, but haven't tested it in deploy tool yet. Good work! 🚀

File: packages/dashmate/configs/schema/configJsonSchema.js (hunk 1 above)
Comment (shumkov) [reply packages/dashmate/configs/schema/configJsonSchema.js:373]: The point is to be able to recover any dashmate state from config. Do you think we need to put config + this file separately?

Comment (shumkov) [reply packages/dashmate/configs/schema/configJsonSchema.js:373]: Removed it from the config

File: packages/dashmate/src/commands/ssl/obtain.js [hunk 3]
```diff
@@ -50,13 +55,20 @@ class ObtainCommand extends ConfigBaseCommand {
 
 ObtainCommand.description = `Obtain SSL certificate
 
-Obtain SSL certificate using ZeroSSL API Key
+Create a new SSL certificate or download already existing one using ZeroSSL provider
+In case if certificate is about to expire (see 'expiration-days' flag) it will be renewed
```
Comment (strophy) [inline packages/dashmate/src/commands/ssl/obtain.js:59]:
```suggestion
Create a new SSL certificate or download an already existing one using ZeroSSL as provider
Certificate will be removed if it is about to expire (see 'expiration-days' flag)
```

**Review by strophy — CHANGES_REQUESTED** [review]
utACK but looks good, one small change to docs grammar

**Pull request merged**